libavfilter/dnn_interface.h

```diff
@@ -59,9 +53,9 @@ typedef enum {
 typedef struct DNNData{
     void *data;
     int width, height, channels;
-    // dt and order together decide the color format
+    // dt and format together decide the color format
+    enum AVPixelFormat format;
```

Comment:
I think you'd better put "format" under "dt". It helps to keep compatibility if user uses offset to access structure.